### PR TITLE
Add editor configurations and instructions for intellij and sublime.

### PR DESCRIPTION
**Pull request by peripateticus**
Adding style guide formatting config files and instructions for Intellij and Sublime.

### CODE REVIEW

**Comment by kellykeene** [comment]
Cool! Thanks, Alex.

**Comment by rhuded** [comment]
works great! thank you!

File: README.md [hunk 1]
```diff
@@ -30,3 +30,17 @@ A developer who would like to propose a change to a code style guide must:
 ### What about code with old formatting?
 
 There's no need to make a specific effort to re-format old code, but if you're modifying a file with outdated formatting, please update what you can while you're in there.
+
+### Editor Configuration
+
+Set up your editor to automatically format code per our style guide:
+
+- **Intellij**
+  - Import [settings](http://git.rei.com/projects/ONLINE/repos/rei.com/browse/config/dev/rei_standard_intellij_settings.jar).
```
Comment (robatron) [inline README.md]: We shouldn't expose internal domains publicly. Please add these files to this repo instead.

**Comment by peripateticus** [comment]
Right, good point.  I'd like to not have the extra step of manually uploading file every time we have an update to the style guide and/or settings file. How about we just indicate the name of the file instead of it's full project path?  Something like 'Import rei_standard_intellij_settings.jar".

**Comment by peripateticus** [comment]
Actually, since this is a public repository, we should include all of the necessary files needed for our style guide.

**Comment by robatron** [comment]
Since these configs apply to all REI software, and they have to be manually imported or referenced anyway, I think it makes sense that they're separate from the REI.com repo, and paired with this one. That way, when a PR for a change is submitted, it can include updates to the config.

**Comment by peripateticus** [comment]
Makes sense, I'll upload these.

**Comment by peripateticus** [comment]
Created new PR for this; closing this one.

**Pull request closed without merging**